src/components/app/app.scss

```diff
@@ -1,3 +1,18 @@
 .app {
-  color: red;
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  height: 100%;
+  width: 100%;
+  &__content {
```

Comment:
please recheck all this statements and leave only those that really have influence in style